src/igraph/drawing/graph.py

```diff
@@ -1276,12 +1277,12 @@ def callback_edge_offset(event):
             else:
                 curved = edge["curved"] if has_curved else False
                 if curved:
-                    aux1 = (2 * x1 + x2) / 3.0 - edge.curved * 0.5 * (y2 - y1), (
```

Comment:
"curved" is not a property of the edge; either edge['curved'] should have been used, or curved as it was assigned to a var.